pkg/model/hetznermodel/loadbalancer.go

```diff
@@ -60,7 +60,7 @@ func (b *LoadBalancerModelBuilder) Build(c *fi.CloudupModelBuilderContext) error
 		},
 	}
 
-	if b.Cluster.UsesNoneDNS() {
+	if b.Cluster.UsesNoneDNS() || b.UseKopsControllerForNodeBootstrap() {
```

Comment:
We probably could just do `b.UseKopsControllerForNodeBootstrap()` at this point, but this seems like the more obvious change.